src/pages/Details.vue

```diff
@@ -155,7 +188,29 @@
             Payment type
           </ul>
         </div>
-        <div class="contentRight"></div>
+        <div class="contentRight">
+          <ul>
+            {{caseDetails.beneficiaryName}} <!--beneficiary-->
+          </ul>
+          <ul>
+            {{caseDetails.beneficiaryPhone}}
+          </ul>
+          <ul>
+            {{caseDetails.beneficiaryEmail}}
+          </ul>
+          <ul>
+            {{caseDetails.beneficiaryPhone}}
+          </ul>
+          <ul>
+            {{caseDetails.beneficiaryHouseholdIncome}}
+          </ul>
+          <ul>
+            {{caseDetails.beneficiaryHouseholdSize}}
+          </ul>
+          <ul>
+            {{caseDetails.beneficiaryPaymentType}}
+          </ul>
+        </div>
         <br />
       </div>
       <div class="sectionHeadingWhite">NOTES</div>
```

Comment:
We are missing the section on related cases

Reply:
updated this section to include related cases